DOCS.md

```diff
@@ -257,9 +258,9 @@ The list of all understood properties is given below.
   const rowMajor = cols => index => [index % cols, Math.floor(index / cols)];
   ```
 
-- The following properties to define the _grid_: `itemSize`, `cellPadding`, `columns`, `rowHeight`, and `cellAspectRatio`
+- The following properties to define the _grid_: `itemSize`, `cellPadding`, `columns`, `rowHeight`, `cellAspectRatio` and `cellSize`.
 
-  One has to at least provide `columns` or `itemSize` to define a grid. If `itemSize` is defined `columns` are ignored. Similarly, when `rowHeight` is defined `cellAspectRatio` is ignored.
+  One has to at least provide `columns`, `itemSize` or `cellSize` to define a grid. If `cellSize` is defined, other 2 properties are ignored. `itemSize` is then defined by `cellSize` and `cellPadding`. If `itemSize` is defined `columns` are ignored. Similarly, when `rowHeight` is defined `cellAspectRatio` is ignored.
```

Comment:
Again `itemSize` should define the size of the item and `cellSize` should define the size of the cell. If `cellSize === null` it should be derived from `itemSize`. That's all.

Reply:
One thing we can implement is that if `itemSize === null` but `cellSize` and `cellPadding` are defined, `itemSize` is derived. But it should be allowed to define `itemSize` and `cellSize`.